source/aggregation.txt

```diff
@@ -80,6 +80,40 @@ The following limitations apply when using aggregation operations:
    </reference/operator/aggregation/graphLookup/>` stage has a strict
    memory limit of 100 megabytes and ignores the ``allowDiskUse`` option.
 
+.. _kotlin-sync-atlas-search-stage:
+
+Pipelines Stages for Atlas Search
+---------------------------------
```

Comment:
S: Consider moving this section after the main aggregation examples on this page. Also consider renaming this section to just `Atlas Search` and then creating a subsection after the paragraph called `Create a Search Query` or something like that?